gorm/transaction.go

```diff
@@ -91,15 +96,19 @@ func (t *Transaction) Rollback() error {
 
 // Commit finishes transaction by calling `*gorm.DB.Commit()`
 // Reset current transaction and returns an error if any.
-func (t *Transaction) Commit() error {
+func (t *Transaction) Commit(ctx context.Context) error {
 	if t.current == nil {
 		return nil
 	}
 	t.mu.Lock()
 	defer t.mu.Unlock()
-
 	t.current.Commit()
 	err := t.current.Error
+	if err == nil && len(t.afterCommitHook) != 0 {
```

Comment:
nitpick: you don't need to check `len` because the `range` statement will work just fine on an empty slice

Reply:
true, I'll modify this.